backend/battle/battles/email.py

```diff
@@ -11,3 +11,14 @@ def result_battle(battle):
             'winner': battle.winner,
         },
     )
+
+
+def send_invite_email(opponent, creator):
+    send_templated_mail(
+        template_name='invite_challenge',
+        from_email=settings.FROM_EMAIL,
+        recipient_list=[opponent],
```

Comment:
Should be `opponent.email` here.
This is only working right now because your `__str__` is `self.email`. You shouldn't rely on that.